### ISSUE TITLE
TypeError: Cannot create a consistent method resolution

### ISSUE THREAD

**Issue by Jerry-py** [andrewthederp/Disgames#4]
When I updated my disgames:

I get this error
```py
TypeError: Cannot create a consistent method resolution
order (MRO) for bases Cog, TicTacToe, Hangman, MadLib, Chess
```

Full error:
```py
PS C:\Users\pushe\Downloads\Bot>
Traceback (most recent call last):
  File "C:\Users\pushe\Downloads\Bot\launcher.py", line 2, in <module>
    from disgames import register_commands
  File "C:\Users\pushe\AppData\Local\Programs\Python\Python310\lib\site-packages\disgames\__init__.py", line 2, in <module>
    from .cog import *
  File "C:\Users\pushe\AppData\Local\Programs\Python\Python310\lib\site-packages\disgames\cog.py", line 11, in <module>
    class Games(commands.Cog, *ALL_GAMES):
  File "C:\Users\pushe\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\cog.py", line 133, in __new__
    new_cls = super().__new__(cls, name, bases, attrs, **kwargs)
TypeError: Cannot create a consistent method resolution
order (MRO) for bases Cog, TicTacToe, Hangman, MadLib, Chess
```

**Comment by Jerry-py** [comment]
Btw 

I also get the error `ImportError`

It's because you forgot to put the chess module in setup.py

**Comment by andrewthederp** [comment]
Should be fixed in version 1.2.4

**Comment by Jerry-py** [comment]
Not fixed yet
```
"discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'Games' object has no attribute 'stockfish_path"]
```

**Comment by MarzaElise** [comment]
should be fixed by now

**Comment by MarzaElise** [comment]
in 1.3.1 release, it *should* be fixed @Jerry-py

**Comment by MarzaElise** [comment]
Fixed in 1.3.1 👍
